ibft/prepare_test.go

```diff
@@ -117,19 +117,19 @@ func TestBatchedPrepareMsgsAndQuorum(t *testing.T) {
 	})
 
 	// test batch
-	res := i.batchedPrepareMsgs(1)
+	res := instance.batchedPrepareMsgs(1)
 	require.Len(t, res, 2)
 	require.Len(t, res[hex.EncodeToString([]byte("value"))], 3)
 	require.Len(t, res[hex.EncodeToString([]byte("value2"))], 1)
 
 	// test valid quorum
-	quorum, tt, n := i.prepareQuorum(1, []byte("value"))
+	quorum, tt, n := instance.prepareQuorum(1, []byte("value"))
```

Comment:
Similarly, I'd use **signedMessages** and **committeeSize** here...
_Note: refactoring code base ofc for consistency_